dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.distributedMapInternal-2.0.feature

```diff
@@ -1,10 +1,10 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=io.openliberty.distributedMapInternal-2.0
+symbolicName=io.openliberty.distributedMapInternal-2.1
```

Comment:
I don't expect to change this feature file at all.  You cannot change the name of the symbolic name without changing the file name to match.  You will get a unit test failure.

Reply:
`visibility/public/distributedMap-1.0/com.ibm.websphere.appserver.distributedMap-1.0.feature` I do expect to change to change the version of the javadoc zip to 2.1 from 2.0.

Reply:
Ok, I've update the javadoc zip and undid the change to the feature file.